test/functional/mempool_updatefromblock.py

```diff
@@ -109,7 +81,7 @@ def transaction_graph_test(self, size, n_tx_to_mine=None, start_input_txid='', e
         self.log.info('Checking descendants/ancestors properties of all of the in-mempool transactions...')
         for k, tx in enumerate(tx_id):
             self.log.debug('Check transaction #{}.'.format(k))
-            entry = self.nodes[0].getmempoolentry(tx)
+            entry = self.nodes[0].getrawmempool(True)[tx['txid']]
```

Comment:
Why is this changed?

Reply:
Changed back to `getmempoolentry`.

I chose `getrawmempool` over `getmempoolentry` to follow some existing functional test files. After a closer look at those two RPCs, I agree the existing `getmempoolentry` here is better as it indicates the "query using a transaction id" intention.